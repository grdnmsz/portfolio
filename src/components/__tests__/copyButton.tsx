import React from 'react'
import renderer from 'react-test-renderer'

import { CopyButton } from '../'

describe('CopyButton', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<CopyButton />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
