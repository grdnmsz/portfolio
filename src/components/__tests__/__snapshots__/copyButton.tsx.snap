// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyButton should render correctly 1`] = `
<button
  className="absolute rounded hover:bg-orange-700 px-2 py-1 right-1 top-1"
  onClick={[Function]}
>
  Copy
</button>
`;
